docs/reference/content/driver-scala/tutorials/ssl.md

```diff
@@ -62,6 +63,33 @@ val settings = MongoClientSettings.builder()
 val client = MongoClients.create(settings)
 ```
 
+### Specify Netty `SslContext` via `NettyStreamFactoryFactory`
+
+If you use the driver with [Netty](https://netty.io/) for network IO,
+you have an option to plug an alternative TLS/SSL protocol implementation provided by Netty.
+
+```java
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.SslProvider;
+```
+
+To instruct the driver to use [`io.netty.handler.ssl.SslContext`]({{< nettyapiref "io/netty/handler/ssl/SslContext.html" >}}),
+use the method
+[`NettyStreamFactoryFactory.Builder.applyToNettySslContext`]({{< apiref "mongodb-driver-core" "com/mongodb/connection/netty/NettyStreamFactoryFactory.Builder.html#applyToNettySslContext(java.util.function.Consumer)" >}}).
+See the documentation of this method for details on which
+[`io.netty.handler.ssl.SslProvider`]({{< nettyapiref "io/netty/handler/ssl/SslProvider.html" >}})s are supported by the driver
+and implications of using them.
+
+```scala
+val settings = MongoClientSettings.builder()
+    .applyToSslSettings((builder: SslSettings.Builder) => builder.enabled(true))
+    .streamFactoryFactory(NettyStreamFactoryFactory.builder()
+        .applyToNettySslContext((builder: SslContextBuilder) => builder.sslProvider(SslProvider.OPENSSL))
+        .build())
+    .build()
+val client = MongoClients.create(settings)
+```
```

Comment:
@rozza Could you please pay attention to my monkey-typing the Scala code here (I did it based on the existing code examples)?

Reply:
Thank you!

Done.